docs/conf/user-interface/index.md

```diff
@@ -0,0 +1,621 @@
+
+| Token Name               | Description                                                       |
+| ------------------------ | ----------------------------------------------------------------- |
+| `--bui-fg-primary`       | It should be used on top of main background surfaces.             |
+| `--bui-fg-secondary`     | It should be used on top of main background surfaces.             |
+| `--bui-fg-link`          | It should be used on top of main background surfaces.             |
+| `--bui-fg-link-hover`    | It should be used on top of main background surfaces.             |
+| `--bui-fg-disabled`      | It should be used on top of main background surfaces.             |
+| `--bui-fg-solid`         | It should be used on top of solid background colors.              |
+| `--bui-fg-tint`          | It should be used on top of tint background colors.               |
+| `--bui-fg-tint-disabled` | It should be used on top of tint background colors when disabled. |
+| `--bui-fg-danger`        | It should be used on top of danger background colors.             |
+| `--bui-fg-warning`       | It should be used on top of warning background colors.            |
+| `--bui-fg-success`       | It should be used on top of success background colors.            |
+
+#### Border colors
+
+These border colors are mostly meant to be used as borders on top of any components with low contrast to help as a separator with the different background colors.
+
+| Token Name              | Description                                         |
+| ----------------------- | --------------------------------------------------- |
+| `--bui-border`          | It should be used on top of `--bui-bg-surface-1`.   |
+| `--bui-border-hover`    | Used when the component is interactive and hovered. |
+| `--bui-border-pressed`  | Used when the component is interactive and hovered. |
+| `--bui-border-disabled` | Used when the component is disabled.                |
+| `--bui-border-danger`   | It should be used on top of `--bui-bg-danger`.      |
+| `--bui-border-warning`  | It should be used on top of `--bui-bg-warning`.     |
+| `--bui-border-success`  | It should be used on top of `--bui-bg-success`.     |
+
+#### Special colors
+
+These colors are used for special purposes like ring, scrollbar, ...
+
+| Token Name              | Description                       |
+| ----------------------- | --------------------------------- |
+| `--bui-ring`            | The color of the ring.            |
+| `--bui-scrollbar`       | The color of the scrollbar.       |
+| `--bui-scrollbar-thumb` | The color of the scrollbar thumb. |
+
+#### Font families
+
+We have two fonts that we use across Backstage UI. The first one is the sans-serif font that we use for the body of the application. The second one is the monospace font that we use for code blocks and tables.
```

Comment:
Let's add "monospace" to the word list here to me make Vale happy: https://github.com/backstage/backstage/blob/master/.github/vale/config/vocabularies/Backstage/accept.txt